packages/widget/src/components/TokenList/TokenList.tsx

```diff
@@ -82,7 +81,11 @@ export const TokenList: FC<TokenListProps> = ({
     !tokenSearchFilter;
 
   return (
-    <Box ref={parentRef} style={{ height, overflow: 'auto' }}>
+    <Box
+      ref={parentRef}
+      style={{ height, overflow: 'auto' }}
+      id="token-list-parent"
```

Comment:
Let's add this id attribute here please to be consistent https://github.com/lifinance/widget/blob/main/packages/widget/src/utils/elements.ts#L1

Reply:
I think can remove these - I've accidently left them in from debug the routes positioning so they aren't really needed